### DISCUSSION TITLE
Using Models from the Zoo on my custom datasets

### DISCUSSION THREAD

**Discussion by AndromathArcanitus** [Project-MONAI/model-zoo#540] [Announcements]
Dear All,

I am new to MONAI and am hoping to use the model lung nodule ct detection from Model Zoo on my own dataset. I have already implemented my DataLoader. So, when I call it, the data is ready. But, when I see the configuration json file, the data is load through

"data_list_file_path": "$@bundle_root + '/LUNA16_datasplit/dataset_fold0.json'",
"dataset_dir": "/datasets/LUNA16_Images_resample",
"train_datalist": "$monai.data.load_decathlon_datalist(@data_list_file_path, is_segmentation=True, data_list_key='training', base_dir=@dataset_dir)",.

Can anyone provide any suggestions for my case? Should I save the data into a folder? But I don't know in what form the data should be saved. Thank you.

**Comment by KumoLiu** [comment]
Hi @AndromathArcanitus, you can change the data path in the configure file if you want to run bundle. Otherwise you could directly using the model weight in your pipeline. Thanks!

**Reply by AndromathArcanitus** [reply]
Hi @KumoLiu , Thank you very much for the timely response. I will certainly try it.

**Reply by AndromathArcanitus** [reply]
Hi @KumoLiu, since I processed my data using different approaches from those for LUNA16, I have already written my DataLoader and loaded the data into the memory (I guess), using, for example,

train_dataset = LungNoduleDataModule(data_folder=data_folder).train_dataloader()

Can I ask how to guide the model to read my data? Thank you.

**Reply by KumoLiu** [reply]
Hi @AndromathArcanitus, if you want to train your pipeline in a pythonic way, you can just follow this [tutorial](https://github.com/Project-MONAI/tutorials/tree/main/bundle/pythonic_usage_guidance) which explore how to access a bundle in Python and use it in your own application.

BTW, you can also find more use case about how to use the bundle in MONAI Zoo [here](https://github.com/Project-MONAI/tutorials/tree/main/model_zoo).

Hope it helps, thanks!

**Reply by AndromathArcanitus** [reply]
It does. Thank you very much.
